README.md

```diff
@@ -46,11 +46,77 @@ Example:
 * "app name" - The application name (default is go-mssqldb)
 * "ApplicationIntent" - Can be given the value "ReadOnly" to initiate a read-only connection to an Availability Group listener.
 
-Example:
+The connection string can be specified in three formats:
 
-```go
-    db, err := sql.Open("mssql", "server=localhost;user id=sa")
-```
+1. `key=value` pairs separated by `;`
+
+    This format doesn't allow characters like `;` within a value. Leading and trailing whitespace is stripped.
+
+    Example:
+
+    ```go
+    db, err := sql.Open("mssql", "server=localhost;user id=sa;password=foobar")
+    ```
+
+1. ODBC format - `key=value` pairs separated by `;`. The string is prefixed with `odbc:`
+
+    This format allows characters like `;` within a value if the value is wrapped in `{}`. Leading and trailing whitespace is retained for wrapped values.
+
+    Example:
+
+    ```go
+    db, err := sql.Open("mssql", "odbc:server=localhost;user id=sa;password={foo;bar}") // password is "foo;bar"
```

Comment:
I think the more common question will also be to specify the instance name `server=localhost\SQLExpress;...` add example with that please.